watcher/kubernetes/pods.go

```diff
@@ -231,3 +269,67 @@ func (pm *PodsManager) watchEvents(ctx context.Context, lg log.Entry, registryDa
 	}()
 
 }
+
+// getFolderLogsPath returns the absolute folder path of the current applyID for pod continers logs
+func (pm *PodsManager) getFolderLogsPath(applyID, podName string) string {
+	return fmt.Sprintf("%s/%s/%s", pm.absoluteLogsPodPath, applyID, podName)
+}
+
+// getPodLogFilePath returns the container file path
+func (pm *PodsManager) getPodLogFilePath(folder, containerName string) string {
+	return fmt.Sprintf("%s/%s.log", folder, containerName)
+}
+
+// podLogs open a container logs steam for getting the STDOUT of container
+func (pm *PodsManager) podLogs(ctx context.Context, lg log.Entry, absoluteFilePath, namespace, podName, containerName string) {
+
+	lgContainer := lg.WithFields(log.Fields{
+		"container_name": containerName,
+		"file_path":      absoluteFilePath,
+	})
+
+	go func() {
+		file, err := os.OpenFile(fmt.Sprintf("%s", absoluteFilePath), os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+		lgContainer.Info("open log stream")
+		podLogOpts := v1.PodLogOptions{
+			Container:  containerName,
+			Follow:     true,
+			Previous:   false,
```

Comment:
does that mean we'll start collecting logs from the point the goroutine started (instead of the beginning)?